Objects/floatobject.c

```diff
@@ -134,6 +134,58 @@ PyFloat_FromDouble(double fval)
     return (PyObject *) op;
 }
 
+#ifdef Py_GIL_DISABLED
+
+PyObject *_PyFloat_FromDouble_ConsumeInputs(_PyStackRef left, _PyStackRef right, double value)
+{
+    PyObject *left_o = PyStackRef_NotDeferred_AsPyObject(left);
+    PyObject *right_o = PyStackRef_NotDeferred_AsPyObject(right);
+    assert(PyLong_Check(left_o));
+    assert(PyLong_Check(right_o));
+    if (Py_REFCNT(left_o) == 1) {
```

Comment:
Is this even safe on nogil builds? I feel that we should just disable it on there for now

Reply:
> Is this even safe on nogil builds?

Was it safe before this PR? I think it was, but it probably isn't fast as `Py_REFCNT` needs to handle biased refcounts.

> I feel that we should just disable it on there for now

Sure. We can re-implement a no-gil version later if we want.